src/Uno.UI/UI/Xaml/Extensions/FontHelper.Android.cs

```diff
@@ -100,7 +106,24 @@ internal partial class FontHelper
 			}
 		}
 
-		private static bool TryLoadFromPath(TypefaceStyle style, string source, out Typeface? typeface, bool encodePath = true)
+		private static string FontStretchToPercentage(FontStretch fontStretch)
+		{
+			return fontStretch switch
+			{
+				FontStretch.UltraCondensed => "50",
+				FontStretch.ExtraCondensed => "62.5",
+				FontStretch.Condensed => "75",
+				FontStretch.SemiCondensed => "87.5",
+				FontStretch.Normal => "100",
+				FontStretch.SemiExpanded => "112.5",
+				FontStretch.Expanded => "125",
+				FontStretch.ExtraExpanded => "150",
```

Comment:
I was just wondering... do you know the reason why the jump here was doubled (25 points instead of 12.5)?

Reply:
@ajpinedam I have no idea. These numbers are from the docs: https://learn.microsoft.com/en-us/uwp/api/windows.ui.text.fontstretch?view=winrt-22621